components/TableColumns/TableColumns.tsx

```diff
@@ -161,20 +155,63 @@ export const leagueDetailsColumns: ColumnDef<LeagueDetailsHeader>[] = [
   },
 ];
 
-export const leagueColumns: ColumnDef<LeagueHeader>[] = [
+export const leagueColumns: ColumnDef<IEntryWithLeague>[] = [
   {
-    accessorKey: 'text',
+    accessorKey: 'leagueName',
     header: 'League Name',
     /**
      * Value of row.
      * @param {object} row - The row data.
      * @param {object} row.row - The row definition
      * @returns {JSX.Element} - The cell component.
      */
-    cell: ({ row }) => <div>{row.getValue('text')}</div>,
+    cell: ({ row }) => <div>{row.getValue('leagueName')}</div>,
   },
   {
-    accessorKey: 'text2',
+    accessorKey: 'survivors',
+
+    /**
+     * Value of row.
+     * @param {object} column - The column data.
+     * @param {object} column.column - The column definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    header: ({ column }): JSX.Element => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          # of Survivors
+          <ChevronsUpDown className="ml-2 h-4 w-4" />
+        </Button>
+      );
+    },
+
+    /**
+     * Value of row.
+     * @param {object} row - The row data.
+     * @param {object} row.row - The row definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: ({ row }): JSX.Element => {
+      const survivors = row.getValue('survivors') as string[];
```

Comment:
Without "as string[]" the {survivors} inside the div returns an error saying that it is unknown.

Reply:
Where is row coming from? Is its type not set when it's returned?

Reply:
So to explain all this:
- cell is a property that defines a function that renders the cell content
- {row} is the object that is already defined by "tanstack" that directly accesses this row so that we can do row.getValue().
         - row.getValue() grabs all the information within 'survivors'
 
- In typescript with this type of function, it doesn't actually know the actual type of this, so we have to concrete it more to let it know it's a string[] and we want the length. I can avoid using the "as string[]" but I would have to take away the ".length" which will actually cause it to show all the userID's

- it's the same for everything else, but I am sure once we optimize the apiFunction, I can tweak the totalEntries and aliveEntries to not have .length